front_end/src/components/onboarding/steps/step_1.tsx

```diff
@@ -0,0 +1,164 @@
+import { sendGAEvent } from "@next/third-parties/google";
+import { useTranslations } from "next-intl";
+import React, { useRef, useState } from "react";
+
+import BinarySlider from "@/app/(main)/questions/[id]/components/forecast_maker/binary_slider";
+import { extractCommunityForecast } from "@/components/onboarding/utils";
+import { OnboardingStep } from "@/types/onboarding";
+
+import Step, { QuestionParagraph } from "./step";
+import Verbal_forecast from "../verbal_forecast";
+
+const Step1: React.FC<OnboardingStep> = ({
+  onNext,
+  topic,
+  posts,
+  setOnboardingState,
+}) => {
+  const t = useTranslations();
+  const [prediction, setPrediction] = useState<number | null>(null);
+  const [activeButton, setActiveButton] = useState<
+    "less" | "about" | "more" | null
+  >(null);
+
+  const endOfModalRef = useRef<HTMLDivElement>(null);
+  // Find related post for Step
+  const post = posts.find((obj) => obj.id === topic.questions[0])!;
```

Comment:
nit: let's not use non-null assertion when possible and handle nullable state gracefully. Same not refers to other steps